src/license/entities/getLicense/activeSeatsGetResult.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  AbstractActiveSeats,
+  ActiveSeatsData,
+} from '../../../common/abstract/abstractActiveSeats';
+
+export type ActiveSeatsGetResultData = ActiveSeatsData;
+
+export class ActiveSeatsGetResult extends AbstractActiveSeats {
```

Comment:
This class adds nothing to `AbstractActiveSeats` so maybe the abstract does not need to exist?

Reply:
done